src/components/BoardsColumn.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="columns-container" :class="{ 'columns-container--sizes': !logo }">
+    <div class="flex gap-6 h-max">
+      <div v-for="column in columns" :key="column.name" class="flex flex-col">
+        <div class="flex items-center gap-2 mb-8">
+          <div
+            class="h-[15px] w-[15px] rounded-full"
+            :class="circleColor(column)"
+          ></div>
+          <p class="text-xs text-gray-400 uppercase">
+            {{ column.name }} ({{ column.tasks.length }})
+          </p>
+        </div>
+        <task-card
+          @change-task-title="(title) => (clickedTitle = title)"
```

Comment:
This function doesn't really change the task title, it changes the title of the clicked task. But it doesn't really matter because this is the part of my hardcoded solutions. I don't even remember why I put this function there in the first place, probably wanted to have some starting point for when I actually do have some real tasks with id.

So, until I store all user data in firebase, there is nothing really that I can do about that (beside deleting this whole function) but I'll remember your suggestion for later on.